io.io

```diff
@@ -0,0 +1,15 @@
+#io language http://iolanguage.org/
+
+FizzBuzz := method(a,
+	if (a % 15 == 0, writeln("FizzBuzz"), 
+		if (a % 3 == 0, writeln("Fizz"), 
+			if (a % 5 == 0, writeln("Buzz"), 
+	writeln(a)
```

Comment:
Is this the indentation style for io? Seems like `writeln(a)` should be indented a level further than the conditions above it.

Reply:
You're right actually, lemme fix that